docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
+The same outcome can be achieved using a plain `Variable` component. Just make sure to write a valid SpEL expression.
 
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_mapmariable1.png "record-variable declaration using a plan Variable component")
 
 &nbsp;
 ## Filter
 
-Filter passes records which satisfy the filtering condition to `true sink`.
+Filters let through records that satisfy a filtering condition.
 
 ![filter graph single](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_filter0.png)
 
-You can additionally define `false sink`. Records from the `source` which meet the filter's condition go to the `true sink`, and others go to the `false sink`.
+You can additionally also define an additional `false sink`. Records from the `source` which meet the filter's conditions are going to be redirected to the `true sink`, while others end up in the `false sink`.
 
 ![filter graph](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_filter1.png)
 
-The Expression field should contain the SpEL expression for the filtering conditions and should produce a boolean value.
+The Expression field should contain a SpEL expression for the filtering conditions and should produce a boolean value.
 
 ![filter window](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_filter2.png)
 
 &nbsp;
 ## Choice
 
-Choice is more advanced variant of filter component - instead of one filtering condition, you can define multiple conditions in some defined order.
+Choice is a more advanced variant of the filter component - instead of one filtering condition, you can define multiple conditions in some defined order.
 It distributes incoming records among output branches in accordance with the filtering conditions configured for those branches.
 
 ![choice graph](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_choice0.png)
 
-Each record from the `source` is tested against the condition defined for outgoing node. If `#input.color` is `blue` record goes to the `blue sink`.  
-If `#input.color` is `green` record goes to the `green sink`. For every other value record goes to the `sink for others` because condition `true` is always true.
-Order of evaluation of conditions is the same as visible in form. You can modify the order using drag & drop functionality.
-Order is also visible on graph in edges description as a number. Be aware that layout button can change displayed order of nodes, but it has no influence on order of evaluation.
+After a record leaves`source`. When it arrives in `choice`, the record value is tested against each of the defined conditions.  If `#input.color` is `blue`, the record ends up in `blue sink`.  
+If `#input.color` is `green`,the record will be sent to the `green sink`. For every other value, the record is sent to `sink for others` because condition `true` is always true.
```

Comment:
```suggestion
If `#input.color` is `green`,the record is sent to the `green sink`. For every other value, the record is sent to `sink for others` because condition `true` is always true.
```